packages/components/text/src/text.styles.ts

```diff
@@ -21,7 +21,7 @@ const truncate = `
 `;
 
 const bold = `
-  font-weight: bold;
+  font-weight: ${designTokens.fontWeightForTextWhenBold}};
```

Comment:
@Rhotimee 
I think this is the reason why `<FieldLabel>` vrts fail
```suggestion
  font-weight: ${designTokens.fontWeightForTextWhenBold};
```

Reply:
Yes, thanks. @CarlosCortizasCT also figured it out.